### PR TITLE
Fix showing of bulk select all the time in inventory

### PR DESCRIPTION
**Pull request by karelhala**

### CODE REVIEW

File: packages/inventory/src/components/table/EntityTableToolbar.js [hunk 1]
```diff
@@ -296,9 +296,11 @@ const EntityTableToolbar = ({
     return <Fragment>
         <PrimaryToolbar
             {...props}
-            bulkSelect={{
-                ...bulkSelect,
-                isDisabled: bulkSelect?.isDisabled || !hasAccess
+            {...bulkSelect && {
```
Comment (rvsia) [inline packages/inventory/src/components/table/EntityTableToolbar.js:299]: why is there a spread operator? :thinking:

Comment (karelhala) [reply packages/inventory/src/components/table/EntityTableToolbar.js:299]: So we don't populate the `bulkSelect` at all if no value is passed. Without it this component could have send `bulkSelect={}` or (which is the point of this PR) `bulkSelect={{ isDisabled: false }}`.

Comment (karelhala) [reply packages/inventory/src/components/table/EntityTableToolbar.js:299]: You can actually see in the failing tests that previsouly it was passing `bulkSelect` with disabled = false and that was causing this exact error.

Comment (rvsia) [reply packages/inventory/src/components/table/EntityTableToolbar.js:299]: Oh.. I am wrong the whole time, I thought it was `(...bulkSelect) && {}` :facepalm: :facepalm: 

I am used to write these conditions as `...(bulkSelect && {})`

Sorry!!! :service_dog:

**Review by rvsia — APPROVED** [review]
snapshots :skull_and_crossbones:

**Pull request merged**